src/sentry/static/sentry/app/views/projectInstall/platform.jsx

```diff
@@ -186,8 +171,7 @@ const ProjectInstallPlatform = createReactClass({
   render() {
     return (
       <div className="install row">
-        <div className="install-content col-md-10">{this.renderBody()}</div>
-        {this.renderSidebar()}
+        <div className="install-content col-md-8 col-md-offset-2">{this.renderBody()}</div>
```

Comment:
The centered content feels odd to me. I personally prefer `col-lg-10` without the offset so it's typically full width and then the max-width at `lg` is a little bigger.